doghouse/server/doghouse.go

```diff
@@ -160,8 +157,35 @@ func (ch *Checker) checkTitle() string {
 }
 
 // https://developer.github.com/v3/checks/runs/#parameters-1
-func (ch *Checker) conclusion() string {
-	switch strings.ToLower(ch.req.Level) {
+func (ch *Checker) conclusion(annotations []*github.CheckRunAnnotation) string {
+	checkResult := "success"
+
+	if ch.req.Level != "" {
+		// Level takes precedence when configured (for backwards compatibility)
+		if len(annotations) > 0 {
+			checkResult = strings.ToLower(ch.req.Level)
+		}
+	} else {
+		levelSeverity := map[string]int{
+			"success": 0,
+			"notice":  1,
+			"warning": 2,
```

Comment:
mhm, is this related to the output of this one?
warnings are `{"ruleId":"@typescript-eslint/no-non-null-assertion","severity":1,"message":"Forbidden non-null assertion.","line":43,"column":19,"nodeType":"TSNonNullExpression","messageId":"noNonNull","endLine":43,"endColumn":54}` severity 1 in my case?
and errors severity 2: 
`  {"ruleId":"prettier/prettier","severity":2,"message":"Delete `·`","line":96,"column":7,"nodeType":null,"messageId":"delete","endLine":96,"endColumn":8,"fix":{"range":[2819,2820],"text":""}}
`

Reply:
No relation. This is just to rank the order of precedence for annotation levels such that error > warning > notice > success.

I can choose a different name to avoid confusion. Anything like rank, value, etc would be fine